### PR TITLE
Changed cursor to wait while waiting for attempt to be deleted.

### PR DESCRIPTION
**Pull request by afzaledx**

### CODE REVIEW

**Comment by afzaledx** [comment]
@muhhshoaib, please review.

File: edx_proctoring/static/proctoring/js/views/proctored_exam_attempt_view.js [hunk 1]
```diff
@@ -174,6 +174,7 @@ var edx = edx || {};
             if (!confirm(gettext('Are you sure you want to remove this student\'s exam attempt?'))) {
                 return;
             }
+            $(this.el).css('cursor', 'wait');
```
Comment (chrisndodge) [inline edx_proctoring/static/proctoring/js/views/proctored_exam_attempt_view.js]: One thing to test is if this style setting will be global to the whole page... try moving the cursor around and see if it changes back to pointer.

If it does change, I think you want to set this style on the whole HTML body. See edx_proctoring/templates/proctored_exam/entrance.html

**Comment by chrisndodge** [comment]
@muhhshoaib can you do an initial review? Thanks. Trying to clear out open PRs...

**Comment by chrisndodge** [comment]
LGTM

let's get a 2nd thumbs up and merge it in...

**Comment by muhhshoaib** [comment]
:+1: . I am going to squash the commits.

**Comment by muhhshoaib** [comment]
@afzaledx squashed the commits. Go ahead and merge it in.

**Pull request merged**